dbt-snowflake/src/dbt/include/snowflake/macros/relations/table/create.sql

```diff
@@ -87,7 +87,6 @@ create or replace {{ transient }}table {{ relation }}
     {%- if contract_config.enforced %}
     {{ get_table_columns_and_constraints() }}
     {%- endif %}
-    {{ optional('cluster by', catalog_relation.cluster_by, '(', '') }}
```

Comment:
We were also applying the cluster key after this create statement via an alter statement, so we only need to remove this clause.